README.md

```diff
@@ -868,6 +868,30 @@ Installs SQL Server on the target node.
 * **[String[]]FailoverClusterIPAddress** _(Write)_: Array of IP Addresses to be assigned to the clustered SQL Server instance. IP addresses must be in [dotted-decimal notation](https://en.wikipedia.org/wiki/Dot-decimal_notation), for example ````10.0.0.100````. If no IP address is specified, uses 'DEFAULT' for this setup parameter.
 * **[String] FailoverClusterNetworkName** _(Write)_: Host name to be assigned to the clustered SQL Server instance.
 
+##### Required Parameters
+
+For configurations that utilize the 'InstallFailoverCluster' action, the following parameters are required (beyond those required for the standalone installation:
+- InstanceName (can be MSSQLSERVER if you want to install a default clustered instance)
```

Comment:
Might have to change to * instead of - as the other lists are using *.
Suggestion is to install VS Code and the markdownlint extension, then you can see this issues directly. :)

C:\projects\xsqlserver\README.md: 874: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 875: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 876: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 877: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 878: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 879: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 880: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 881: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 882: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 883: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 884: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 885: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 886: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 887: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 888: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 889: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 890: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 891: MD004 Unordered list style
C:\projects\xsqlserver\README.md: 893: MD012 Multiple consecutive blank lines
C:\projects\xsqlserver\README.md: 894: MD012 Multiple consecutive blank lines
C:\projects\xsqlserver\README.md: 874: MD032 Lists should be surrounded by blank lines